nbdev/cli.py

```diff
@@ -270,4 +271,64 @@ def nbdev_install_git_hooks():
         f.write("""**/*.ipynb filter=clean-nbs
 **/*.ipynb diff=ipynb
 """
-               )
\ No newline at end of file
+               )
+
+# Cell
+from getpass import getuser
+
+# Cell
+_template_git_repo = "git@github.com:fastai/nbdev_template"
+
+# Cell
+def create_template_config(path, lib_name):
+    "Create a template config file in `path` for `libname` using the current users git config if it exists."
+
+    git_config = ConfigParser()
+    git_config.read(Path("~/.gitconfig").expanduser())
+
+    if 'user' in git_config:
+        create_config(lib_name, getuser(), path,
+                  author = git_config.get('user', 'name'),
+                  author_email = git_config.get('user', 'email'),
+                  copyright = git_config.get('user', 'name'))
+
+
+# Cell
+@call_parse
+def nbdev_new(name: Param("A directory to create the project in", str)):
+    "Create a new nbdev project with a given name."
+
+    path = Path(f"./{name}").absolute()
+
+    if path.is_dir():
+        print(f"Directory {path} already exists. Aborting.")
+        return
+
+    print(f"Creating a new nbdev project {name}:")
+
+    try:
+        print("Cloning the template repo")
+        subprocess.run(f"git clone {_template_git_repo} {path}".split(), check=True, timeout=5000)
+
+        print("Initializing your git repository")
+        shutil.rmtree(path/".git")
+        subprocess.run("git init".split(), cwd=path, check=True)
+        subprocess.run("git add .".split(), cwd=path, check=True)
+        subprocess.run("git commit -am \"Initial\"".split(), cwd=path, check=True)
+
+        print("Settings up your settings.ini")
+
+        settings_path = path/"settings.ini"
+        example_settings_path = path/"settings.ini.example"
+
+        if settings_path.is_file():
+            print("Moving an existing settings.ini to settings.ini.example")
+            settings_path.rename(example_settings_path)
+
+        create_template_config(path, name)
+
+        print("Done!")
```

Comment:
Maybe say something more descriptive. e.g. "Completed creating repo from template. Please edit settings.ini now, and then then nbdev_build_lib".